Objects/abstract.c

```diff
@@ -1714,6 +1715,46 @@ PyNumber_ToBase(PyObject *n, int base)
     return res;
 }
 
+typedef struct {
+    const int slot;
+    const char name[3];
+    const int islot;
+    const char iname[4];
+} nb_info;
```

Comment:
This seems to have been merged back into a single table. 
See https://github.com/python/cpython/pull/29255#discussion_r739804796